ts/features/itwallet/common/store/actions/preferences.ts

```diff
@@ -16,8 +16,13 @@ export const itwUnflagCredentialAsRequested = createStandardAction(
   "ITW_UNFLAG_CREDENTIAL_AS_REQUESTED"
 )<string>();
 
+export const itwReviewRequested = createStandardAction("ITW_REVIEW_REQUESTED")<
+  boolean | undefined
+>();
```

Comment:
For consistency, I think it's more appropriate. Also I don't think we need to pass `undefined`

```suggestion
export const itwSetReviewRequested = createStandardAction("ITW_SET_REVIEW_REQUESTED")<
  boolean
>();
```

Reply:
Yes, you're right. It's better to use the action name as you suggested, and indeed, `undefined` is not necessary.
Do you think it's better to use `ITW_SET_PENDING_REVIEW` as the action name for consistency with the `state` name `isPendingReview`?